node/pkg/logscribe/api/controller.go

```diff
@@ -0,0 +1,40 @@
+package api
+
+import (
+	"encoding/json"
+
+	"bisonai.com/orakl/node/pkg/db"
+	"github.com/gofiber/fiber/v2"
+	"github.com/rs/zerolog/log"
+)
+
+type LogInsertModel struct {
+	Service   string          `db:"service" json:"service"`
+	Timestamp string          `db:"timestamp" json:"timestamp"`
+	Level     int             `db:"level" json:"level"`
+	Message   string          `db:"message" json:"message"`
+	Fields    json.RawMessage `db:"fields" json:"fields"`
+}
+
+func insertLogs(c *fiber.Ctx) error {
```

Comment:
how about making it return response quickly and do all the inserts from background?

just sending all the entries into channel and run bulkCopy in a regular basis

Reply:
sounds good!